tools/src/drawbridge.rs

```diff
@@ -37,21 +38,15 @@ pub fn build_packet<'a>(
         return Err(DoesNoteExist.into());
     }
 
-    // Init tempsepc
-    let mut timestamp = libc::timespec {
-        tv_sec: 0,
-        tv_nsec: 0,
-    };
-
-    // get current timestamp
-    unsafe {
-        libc::clock_gettime(libc::CLOCK_REALTIME, &mut timestamp);
-    }
+    let secs = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap()
```

Comment:
I think this should probably just handle the potential error instead of a call to `unwrap`.

```rust
let secs = SystemTime::now()
        .duration_since(UNIX_EPOCH)?
        .as_secs();
```

Reply:
Hi @landhb , modify to use `?` to handle potential error instead of a call to `unwrap`, please review.